pycaret/internal/utils.py

```diff
@@ -32,6 +32,14 @@ class SeasonalPeriod(IntEnum):
     Y = 1 #year
 
 
+class TSModelTypes(Enum):
+    baseline = 'baseline'
```

Comment:
Looks good overall. 

Sorry for being picky. General convention is to keep the first one UPPERCASE since it is a constant. If it is not too much work, could you change it to make it consistent. Thanks!

https://docs.python.org/3/library/enum.html